### PR TITLE
Feature/setup as shell

### LINKED ISSUE #120: Create install/update script
scrip should be able to follow installation steps with ability to parametrise configuration 
It should receive URL param to patch `REACT_APP_API_URL` and `APP_FRONTEND_URL` in `.env` file 
and params for api/ui/migration image  in `docker-compose.yml`
if possible add checks if docker and docker-compose are installed

## Install scenario
download default `docker-compose.yml`
download default `.env`
patch `.env`
start

### Update scenario
stop containers 
patch `docker-compose.yml`
start

### PR DESCRIPTION
**Pull request by dcrowe**
resolves #120 

This has a couple of changes from Alex's work:
* supports more cmd-line arguments, including long arguments
* detaches docker-compose gracefully
* generates a vrt.json file

It also has a downside, it cannot be re-run once the db directory exists. The vrt.json generation code doesn't work then. Any ideas on how to fix this?

Tested on macOS.
Will test on ubuntu tomorrow.
Do you have access to WSL?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,41 @@ Service receives images, performs pixel by pixel comparison with it’s previous
...
 
 ## Set up
+
+### Linux, macOS, WSL
+
+1. Install [Docker](https://docs.docker.com/get-docker/)
+2. Download the installation script
+
+```
+curl https://raw.githubusercontent.com/Visual-Regression-Tracker/Visual-Regression-Tracker/master/vrt-install.sh
```
Comment (pashidlos) [inline README.md:33]: add `-o vrt-install.sh` otserwise it's not saved as file

File: vrt-install.sh [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+echo " "
+echo Waiting for VRT to start
+docker-compose logs --follow | grep -q -m 1 'The Api key is'
+API_KEY=`docker-compose logs | grep 'The Api key is' | sed -e 's,^.*: ,,g'`
+
+echo " "
+echo Populating vrt.json
+cat << EOF > vrt.json
+{
+    "apiUrl": "${REACT_APP_API_URL}",
```
Comment (pashidlos) [inline vrt-install.sh:120]: `apiUrl` is not populated

File: vrt-install.sh [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    cat <<EOF
+Installs the Visual Regression Tracker
+
+Usage: ./vrt-install.sh
+
+Arguments:
+    -h | --help
+    -a | --frontend-url <url>   Set the Front-end url. Default: http://localhost:8080
+    -r | --backend-url <url>    Set the API url. Default: http://localhost:4200
+    -i | --images-folder <dir>  Set the location of the image folder.
```
Comment (pashidlos) [inline vrt-install.sh:27]: please, remove this as param 
found that it works not as I intended and value from `env` is defining folder name inside container 
we gonna use `imageUploads` as hardcoded value and I'll remove this from `env` file

Comment (pashidlos) [reply vrt-install.sh:27]: will do it in scope of 
https://github.com/Visual-Regression-Tracker/Visual-Regression-Tracker/issues/160

**Comment by pashidlos** [comment]
> It also has a downside, it cannot be re-run once the db directory exists. The vrt.json generation code doesn't work then. Any ideas on how to fix this?

for me it's fine if script works only once and then just calls `docker-compose up` in case all files are there
config file will be posible to copy from frontend on next release

**Comment by pashidlos** [comment]
Changed target branch to `release-4.7.0` in order to gather all changes for regression testing before release

File: vrt-install.sh (hunk 3 above)
Comment (dcrowe) [reply vrt-install.sh:27]: 👍

File: README.md (hunk 1 above)
Comment (dcrowe) [reply README.md:33]: fixed

File: vrt-install.sh (hunk 2 above)
Comment (dcrowe) [reply vrt-install.sh:120]: Hmm. I may have forgotten to push a commit.

**Comment by dcrowe** [comment]
Tested now on:
* Ubuntu 18.04.5

**Comment by pashidlos** [comment]
works as expected!

**Pull request merged**